### PR TITLE
machine docs: removed references to AMI per @nathanleclaire comments

### PR DESCRIPTION
**Pull request by londoncalling**
Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
LGTM

**Comment by londoncalling** [comment]
@nathanleclaire @jeanlaurent @dgageot when someone gets a chance, can you review/merge? Should be good to go. Thanks!

File: docs/examples/aws.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Follow along with this example to create a Dockerized <a href="https://aws.amazo
 
 ### Step 4. Use Machine to remove the instance
 
-To remove a AMI instance and all of its containers and images, first stop the machine, then use `docker-machine rm`:
+To remove a instance and all of its containers and images, first stop the machine, then use `docker-machine rm`:
```
Comment (nathanleclaire) [inline docs/examples/aws.md]: Nit: "an" instance?

**Comment by nathanleclaire** [comment]
Grammar nitpick -- but LGTM

File: docs/examples/aws.md (hunk 1 above)
Comment (londoncalling) [inline docs/examples/aws.md]: oh man, I'm lame ... one second

**Comment by londoncalling** [comment]
@nathanleclaire Fixed grammar and made sure the error was not propagated elsewhere. Should be good-to-go after build, please merge when you get a chance. Thanks!

**Comment by nathanleclaire** [comment]
LGTM -- thanks @londoncalling !

**Pull request merged**